libs/postgres_ffi/build.rs

```diff
@@ -57,81 +57,97 @@ fn main() {
     };
     // Currently, we only expect to find PostgreSQL v14 sources, in "pg_install/v14". In the
     // future, we will run this for all supported PostgreSQL versions.
-    pg_install_dir.push("v14");
 
-    if pg_install_dir.is_relative() {
-        let cwd = env::current_dir().unwrap();
-        pg_install_dir = cwd.join("..").join("..").join(pg_install_dir);
-    }
+    for pg_version in vec!["v14", "v15"] {
+        pg_install_dir.push(pg_version);
+        if pg_install_dir.is_relative() {
+            let cwd = env::current_dir().unwrap();
+            pg_install_dir = cwd.join("..").join("..").join(pg_install_dir);
+        }
 
-    let pg_config_bin = pg_install_dir.join("bin").join("pg_config");
-    let inc_server_path: String = if pg_config_bin.exists() {
-        let output = Command::new(pg_config_bin)
-            .arg("--includedir-server")
-            .output()
-            .expect("failed to execute `pg_config --includedir-server`");
+        let pg_config_bin = pg_install_dir
+            .join("bin")
+            .join("pg_config")
+            .join(pg_version);
```

Comment:
ISTM this will try to find the `pg_config` binary as `pg_install/bin/pg_config/v14`. Shouldn't it be `pg_install/v14/bin/pg_config`?